sql/admin.md

```diff
@@ -139,6 +139,46 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...
+    | 43     | test    | t1         | add index     | public               | 32        | 37       | 6         | 2019-01-10 12:35:13.66 +0800 CST  | synced        |
+    | 42     | test    | t1         | drop index    | none                 | 32        | 37       | 0         | 2019-01-10 12:34:35.204 +0800 CST | synced        |
+    | 41     | test    | t1         | add index     | public               | 32        | 37       | 0         | 2019-01-10 12:33:22.62 +0800 CST  | synced        |
+    | 40     | test    | t1         | drop column   | none                 | 32        | 37       | 0         | 2019-01-10 12:33:08.212 +0800 CST | synced        |
+    | 39     | test    | t1         | add column    | public               | 32        | 37       | 0         | 2019-01-10 12:32:55.42 +0800 CST  | synced        |
+    | 38     | test    | t1         | create table  | public               | 32        | 37       | 0         | 2019-01-10 12:32:41.956 +0800 CST | synced        |
+    | 36     | test    |            | drop table    | none                 | 32        | 34       | 0         | 2019-01-10 11:29:59.982 +0800 CST | synced        |
+    | 35     | test    |            | create table  | public               | 32        | 34       | 0         | 2019-01-10 11:29:40.741 +0800 CST | synced        |
+    | 33     | test    |            | create schema | public               | 32        | 0        | 0         | 2019-01-10 11:29:22.813 +0800 CST | synced        |
+    +--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
+    ```
+
+    * `JOB_ID`: each DDL operation corresponds to one DDL job. `JOB_ID` is globally unique.
+    * `DB_NAME`: the name of the database on which the DDL operations are performed.
+    * `TABLE_NAME`: the name of the table on which the DDL operations are performed.
+    * `JOB_TYPE`: the type of the DDL operations.
+    * `SCHEMA_STATE`: the current state of the schema. If the `JOB_TYPE` is `add index`, it's the state of the index; if the `JOB_TYPE` is `add column`, it's the state of the column; if the `JOB_TYPE` is `create table`, it's the state of the table. The common states include:
+        * `none`: it indicates not existing. When the drop or create operation fails and rolls back, it usually becomes the `none` state.
+        * `delete only`, `write only`, `delete reorganization`, `write reorganization`: these four states are intermediate states. For details, see the paper [Online, Asynchronous Schema Change in F1](http://static.googleusercontent.com/media/research.google.com/zh-CN//pubs/archive/41376.pdf). These states cannot be seen in common operations, because the conversion from the intermediate states is so quick. Only in `add index` operations, you can see the `write reorganization` state, which means that the index data is being added. 
```

Comment:
```suggestion
        * `delete only`, `write only`, `delete reorganization`, `write reorganization`: these four states are intermediate states. For details, see the paper [Online, Asynchronous Schema Change in F1](http://static.googleusercontent.com/media/research.google.com/zh-CN//pubs/archive/41376.pdf). These states cannot be seen in common operations, because the conversion from the intermediate states is so quick. You can see the `write reorganization` state only in `add index` operations, which means that the index data is being added.
```